src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int NUMBER_OF_RANGE = 100;
+    private Ball ball;
+
+    public Lottery() {
+        this.ball = new Ball();
+    }
+
+    public Ball getRandomBall() {
```

Comment:
your ball that you are getting in the main method is not even set or changed here